doctest.sh

```diff
@@ -22,10 +22,15 @@ if [[ "$OS" =~ Windows ]]; then
     mingw32-make
     jou="$PWD/jou.exe"
 else
-    make
+    eval $(which gmake || which make)
     jou="$PWD/jou"
 fi
 
+DIFF=$(which gdiff || which diff)
+if [[ x"$($DIFF --help | grep -- --color)" != x ]]; then
```

Comment:
You could maybe use `grep -q` if it's portable enough. Not sure.

Reply:
Ah, I did have a little feeling that something like that must exist. `-q` it is, indeed!